### PR TITLE
Identicon support

### PR DESCRIPTION
**Pull request by Daviddt17**
I added the angular-identicon module for use of identicons. Identicons will be generated from usernames. After login, your identicon will appear in the upper-right corner of the toolbar. Your identicon will also appear on the account page. Support for identicons elsewhere on the site is ready to go as long as we have access to usernames.

@Mctalian @YashdalfTheGray @davethenipper

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -1,31 +1,32 @@
...
+    "angular-aria": "1.4.7",
+    "angular-mocks": "1.4.7",
+    "angular-animate": "1.4.7",
+    "angular-material": "0.11.2",
+    "angular-ui-router": "0.2.15",
+    "lodash": "3.10.0",
+    "moment": "2.10.3",
+    "event-source-polyfill": "0.0.6",
+    "identicon": "*",
+    "angular-identicon": "~0.1.17"
```
Comment (davethenipper) [inline bower.json]: angular-identicon 0.1.17 looks to be the only change in bower.json besides spacing.  Correct?

Comment (Daviddt17) [inline bower.json]: Yeah. Not sure why it decided to mark everything as an add as opposed to just that line.

**Comment by davethenipper** [comment]
I also updated the initial comment to tag some reviewers.

File: bower.json (hunk 1 above)
Comment (davethenipper) [inline bower.json]: Could this be the indentation?

Comment (Mctalian) [inline bower.json]: Looks like 2 spaces instead of 4?

Comment (Mctalian) [inline bower.json]: Also, can you take out the ~? We want to specify exact versions so everyone's build is the same

**Comment by YashdalfTheGray** [comment]
David, did you use `npm install --save` to pull in the library? I think I can explain the spacing change.

**Comment by Daviddt17** [comment]
@YashdalfTheGray If I remember correctly, I used `bower install angular-identicon --save`.

**Comment by Mctalian** [comment]
That'll do it.

**Comment by YashdalfTheGray** [comment]
Yup, that'll rewrite the entire bower file basically. And the json standard is 2 spaces anyway.

**Comment by YashdalfTheGray** [comment]
Can we merge this? It's approved from my side.

**Comment by Mctalian** [comment]
Haven't had a chance to look at it. Will be doing a lot of that work
shortly.

**Comment by Mctalian** [comment]
I approve pending the ~ removal from angular-identicon in bower.json, and we might as well remove the identicon line above it.

**Comment by YashdalfTheGray** [comment]
Ooh, good catch.

**Comment by Daviddt17** [comment]
I think identicon is a dependency of angular-identicon. So that line needs to stay? Also, do I make the changes you guys suggested and make another pull? Or do you guys handle it upon accepting?

**Comment by Mctalian** [comment]
If identicon is a dependency of angular-identicon, it will be handled in
angular-identicon's own bower.json.

And if you make changes, they will automatically update here. No need to
open another PR. As long as Github can see your updates, the PR will see
them.

**Comment by Daviddt17** [comment]
Will identicon still find its way to the lib folder? I'll make the chages and push.

**Comment by Mctalian** [comment]
If you need to reference identicon in the library folder anywhere in the
code, then leave it in. It's not hurting anything right now either way.

File: src/index.html [hunk 2]
```diff
@@ -18,6 +18,10 @@
     <script src="lib/angular-mocks/angular-mocks.js"></script>
     <script src="lib/angular-material/angular-material.js"></script>
     <script src="lib/angular-aria/angular-aria.js"></script>
+    <script src="lib/angular-identicon/dist/angular-identicon.js"></script>
+    <script src="lib/angular-identicon/src/md5.js"></script>
+    <script src="lib/identicon/identicon.js"></script>
```
Comment (Mctalian) [inline src/index.html]: It looks like I was wrong and you will probably need identicon in the bower.json file.

**Comment by Daviddt17** [comment]
Added it back to the list!

**Comment by Mctalian** [comment]
Approved. Looks like you will have some merge conflicts to handle, though.

**Pull request merged**